ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -338,7 +341,12 @@ object KeyValueConsumption {
               Status.of(
                 Code.ALREADY_EXISTS.value,
                 "Duplicate commands",
-                Seq.empty,
+                Seq(
+                  AnyProto.pack[ErrorInfo](
+                    // the definite answer is false, as the rank-based deduplication is not yet implemented
+                    ErrorInfo(metadata = Map(GrpcStatuses.DefiniteAnswerKey -> "false"))
```

Comment:
This must take the value from `transactionRejectionEntry.definiteAnswer`.

Reply:
You're right, no idea why I missed it.